packages/ihe-gateway-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/ihe-gateway-sdk",
-  "version": "0.1.11",
+  "version": "0.1.12-alpha.1",
```

Comment:
Defs NABD now, but just bringing attention that a change like this would warrant a major update, but since we didn't release v1 yet it could be minor.